tools/cdk-release/lib/types.ts

```diff
@@ -9,30 +9,16 @@ export interface Lifecycles {
   tag?: string;
 }
 
-type LifecyclesSkip = {
+export type LifecyclesSkip = {
   [key in keyof Lifecycles]: boolean;
 }
 
-/* ****** Updaters ******** */
-
-export interface UpdaterModule {
-  isPrivate?: (contents: string) => string | boolean | null | undefined;
-  readVersion(contents: string): string;
-  writeVersion(contents: string, version: string): string;
-}
-
-export interface ArgUpdater {
-  filename: string;
-  type?: string;
-  updater?: UpdaterModule | string;
+export interface Version {
```

Comment:
I would quibble with the naming here: `Version` is-a `tuple of 2 strings`, each of which is a full version string by itself... ?

Call it `Versions` at least?

Reply:
Also perhaps `stableVersion` ?

Reply:
`Versions` is fine. I _originally_ went with `stableVersion`, but wanted _slightly_ easier reading/writing to the existing versions file. I suppose it's technically more correct not to. I'll adjust. :)